core/src/main/java/tc/oc/pgm/flag/LegacyFlagBeamMatchModule.java

```diff
@@ -188,7 +204,8 @@ public void run() {
       this.flag = flag;
       this.bukkit = bukkit;
       this.segments =
-          range(0, 32)
+          range(0, 100) // ~100 blocks is the height which the particles appear to be reasonably
```

Comment:
Having 100 zombie entities may be a bit too much on the client, and i doubt in any reasonable scenario this will actually be a factor that limits/helps one player or another, maybe 64 because 32 does look a bit on the lower side?

Reply:
I get what you mean, however at times a beam of 32 zombies can sometimes be a bit too short for maps which have underground play areas, and these beams at times sticked out not as high as the particle beams.

Reply:
Yeah, that's why i suggested maybe make it 64, i'm hoping that should be more than enough, 32 *can be* short at times

Reply:
64 is okay, but I believe the official flag beams(particle beams( are supposed to run 48 blocks tall if I'm reading https://github.com/PGMDev/PGM/blob/7bd6dd2dcb628f7fdce00c8bd378faf6a9c5d61d/core/src/main/java/tc/oc/pgm/flag/state/Spawned.java#L62 correctly, but in-game these particles exist no matter how high you go, they just decrease in amount the higher you go.

Reply:
Some particles aren't an exact science, this ones spawn 56 blocks above the flag, and go "roughly" 24 blocks up/down, this all has randomness to it and the flags themselves will spawn randomly and fall too.
Technically speaking, particles should spawn from 32 blocks above the flag to 80 blocks above the flag, but they fall and cover a larger area

Reply:
Yea, the particles are weird since they spawn and fall like that. Even though they are supposed to spawn anywhere from 32 to 80 blocks above, theres a weird thing where a few particles still spawn about 130 blocks above the flag (in my testing flag spawns at y=20, particles appear to stop completely above y=150)

Reply:
Changed to 64.